src/main/java/com/minecolonies/core/colony/buildings/modules/RestaurantMenuModule.java

```diff
@@ -205,7 +207,11 @@ public void deserializeNBT(final CompoundTag compound)
         final ListTag minimumStockTagList = compound.getList(TAG_MENU, Tag.TAG_COMPOUND);
         for (int i = 0; i < minimumStockTagList.size(); i++)
         {
-            menu.add(new ItemStorage(ItemStack.of(minimumStockTagList.getCompound(i))));
+            final ItemStack itemStack = ItemStack.of(minimumStockTagList.getCompound(i));
+            if (ISFOOD.test(itemStack) || itemStack.getItem() instanceof IMinecoloniesFoodItem)
```

Comment:
Don't we have somewhere a util call for this?

Reply:
Not that I could find, no

Reply:
FoodUtils.EDIBLE is what we use elsewhere, dont think we need the instanceof. If if cannot be eaten it should not be on the menu